### PR TITLE
Remove k command and use dnx instead

### PR DESCRIPTION
**Pull request by ChengTian**
https://github.com/aspnet/dnx/issues/1473
@davidfowl @Praburaj @muratg

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,17 +14,17 @@ This project is part of ASP.NET 5.0. You can find samples, documentation and get
 ## Run on WebListener/Kestrel:
 * Open a command prompt and cd `\src\MusicStore\`.
 * **[WebListener]:**
-	4. Run `k web` (Application started at URL **http://localhost:5002/**).
+	4. Run `dnx . web` (Application started at URL **http://localhost:5002/**).
```
Comment (Praburaj) [inline README.md]: why does it have a `.`?

Comment (ChengTian) [inline README.md]: It's appbase. We are not simply renaming `k.cmd` to `dnx.cmd`, we are discarding `k.cmd` and use `dnx.exe`, which is invoked by `k.cmd`. `dnx.exe` has different usage.

Comment (hishamco) [inline README.md]: @Praburaj we should use dnx . web and dnx .test to run and test the current application

Comment (Praburaj) [inline README.md]: sounds good. Thanks!

**Pull request merged**